CHANGES.md

```diff
@@ -146,7 +147,7 @@
 ## Known Issues
 
 * [Python] If you are using the official Apache Beam Python containers for version 2.62.0, be aware that they include NumPy version 1.26.4. It is strongly recommended that you explicitly specify numpy==1.26.4 in your project's dependency list. ([#33639](https://github.com/apache/beam/issues/33639)).
-
+* [Dataflow Streaming Appliance] Commits fail with KeyCommitTooLargeException when a key outputs >180MB of results. Bug affects versions 2.60.0 to 2.62.0, fix will be released with 2.63.0.  [#33588](https://github.com/apache/beam/issues/33588).
```

Comment:
Should you mention how to fix here (ie use different version or use streaming engine?)

Reply:
modified.